.bashrc.template

```diff
@@ -1 +1,2 @@
 alias config-rebuild="pushd . && cd <<CONFIG_REPO_PATH>> && ./build_config.sh && sudo cp generated/configuration.nix /etc/nixos/ && sudo nixos-rebuild switch && popd"
+export XMONAD_CONFIG_DIR=<<CONFIG_REPO_PATH>>
```

Comment:
Originally used XDG_CONFIG_HOME, but that also messed up my git configs. I'm going to exclusively add program-specific env vars to avoid bonking other configs that Nix has set up for me.